src/components/ExecutePassButton.js

```diff
@@ -1,14 +1,10 @@
 import React from 'react';
+import { useContext } from 'react';
+import { GameContext } from '../context/GameContext.js';
 import Card from './Card';
 
-export default function ExecutePassButton({
-  passCard,
-  setFrom,
-  from,
-  to,
-  selectedCard,
-  setSelectedCard,
-}) {
+export default function ExecutePassButton({ passCard }) {
+  const { selectedCard, setSelectedCard, setFrom, from, to } = useContext(GameContext);
   return (
     <div className="execute-button" onClick={() => passCard(selectedCard)}>
       Pass <Card card={selectedCard} setSelectedCard={setSelectedCard} setFrom={setFrom} /> from{' '}
```

Comment:
You don't need setSelectedCard because you are getting it from context in the Card component